vendor/bat-native-ads/src/bat/ads/internal/base/url/url_util.cc

```diff
@@ -84,14 +84,24 @@ bool MatchUrlPattern(const GURL& url, const std::string& pattern) {
   return base::MatchPattern(url.spec(), pattern);
 }
 
-bool SameDomainOrHost(const GURL& lhs, const GURL& rhs) {
+bool SameETLDPlusOneForUrl(const GURL& lhs, const GURL& rhs) {
+  return lhs.host() == rhs.host();
```

Comment:
the hostname may differ from eTLD+1

Reply:
using host() I tested the following:

GURL("https://www.bar.com/").host():
Output: www.bar.com

GURL("https://foo.bar.com/").host()
Output: foo.bar.com

GURL("https://baz.foo.bar.com/").host()
Output: baz.foo.bar.com

GURL("https://bar.com/").host()
Output: bar.com

Reply:
Which is what we need for this use case. Do you have an example of a .host() failing, please?

Reply:
Good catch. Fixed